packages/api/src/services/security/test/security-context.service.spec.ts

```diff
@@ -0,0 +1,39 @@
+import {SecurityContextService} from '../security-context.service';
+import {DeniedPermissions} from '../../../models/security/denied-permissions';
+
+describe('SecurityContextService', () => {
+  let securityContextService: SecurityContextService;
+
+  beforeEach(() => {
+    securityContextService = new SecurityContextService();
+  });
+
+  test('Should call onUserPagePermissionChange callback with correct denied routes', () => {
+    const onPermissionChangedCallBackFunction = jest.fn();
+    const newPermissions = {'/testPage': true} as DeniedPermissions;
```

Comment:
Give the suggestion for this class, this can become for example:
```
const newPermissions = new DeniedPermissions();
newPermissions.denyPermission('/testPage');
```

Reply:
done